### ISSUE TITLE
Set up rollup to build UMD and es module

### ISSUE THREAD

**Issue by mfogel** [mfogel/polygon-clipping#18]
The es module should be set to `package.module`, the UMD to `package.main`

Once this is done, the source code can be adjusted to use es6 imports/exports.

**Comment by barnabas-avalara** [comment]
Thank you for your work on this. I think this would enable using polygon-clipping as a drop-in replacement of w8r/martinez for turfjs functions in the browser, at least for a project that uses Babel to transpile es6. (Don't you just love JavaScript?)

For context, I'm trying to replace turf-union to see if this fork corrects some known bugs, like so:
```javascript
import clipping from 'polygon-clipping'
import { getGeom } from '@turf/invariant'
import { multiPolygon, polygon } from '@turf/helpers'

export function union (polygon1, polygon2, options = {properties: {}}) {
  const coords1 = getGeom(polygon1).coordinates
  const coords2 = getGeom(polygon2).coordinates

  const unioned = clipping.union(coords1, coords2)
  if (unioned.length === 0) return null
  if (unioned.length === 1) return polygon(unioned[0], options.properties)
  return multiPolygon(unioned, options.properties)
}
```

But sadly, invoking it returns this:
```
TypeError: SplayTree is not a constructor
    at new SweepLine (sweep-line.js?abd0:18)
    at doIt (index.js?e521:40)
    at Object.union (main.js?4d96:5)
```

For reference, this is polygon-clipping@0.6.1 and splaytree@0.1.4. You might be interested to know that w8r/splay-tree was upgraded to v2.0 about a week and a half ago, possibly related.

Cheers!

**Comment by mfogel** [comment]
Thanks for the tip on the splaytree update. I updated that dependency just now - it's a shot in the dark but if you want to try running what's on master now it's possible that it might fix the problem. In any case, I hope to have this addressed within a few weeks and a new version pushed to npm.

**Comment by barnabas-avalara** [comment]
Thanks for the splaytree update. I tried it out using your master branch, didn't work, same problem.

**Comment by mfogel** [comment]
Addressed in #20
